components/meta-packages/sunray-essential/sunray-essential.p5m

```diff
@@ -0,0 +1,107 @@
+#
+# This file and its contents are supplied under the terms of the
+# Common Development and Distribution License ("CDDL)". You may
+# only use this file in accordance with the terms of the CDDL.
+#
+# A full copy of the text of the CDDL should have accompanied this
+# source. A copy of the CDDL is also available via the Internet at
+# http://www.illumos.org/license/CDDL.
+#
+
+#
+# Copyright 2025, Carsten Grzemba. All rights reserved.
+#
+
+set name=pkg.fmri value=pkg:/$(COMPONENT_FMRI)@$(IPS_COMPONENT_VERSION),$(BUILD_VERSION)
+set name=pkg.human-version value=$(HUMAN_VERSION)
+set name=pkg.summary value="$(COMPONENT_SUMMARY)"
+set name=info.classification value="$(COMPONENT_CLASSIFICATION)"
+set name=org.opensolaris.consolidation value=$(CONSOLIDATION)
+
+# graphical desktop
+depend fmri=mate_install type=require
+
+# DHCP service
+depend fmri=service/network/dhcp/isc-dhcp type=require
+# empty package only for satisfy installation requirements
+# SRSS on Solaris uses the old Sun DHCP server but on OI we use ISC-DHCP
+depend fmri=service/network/dhcp type=require
+
+# SUNWutkio-libs package dependency, empty packages only
+# for satisfy installation requirements
+depend fmri=consolidation/X/X-incorporation type=require
+depend fmri=compatibility/packages/SUNWxwplt type=require
+
+# for Sun JRE
+depend type=require fmri=__TBD pkg.debug.depend.file=usr/lib/libCrun.so.1
+
+# SUNWut configure requirements rpm2cpio, see docs.openindiana.org
+depend type=require fmri=__TBD pkg.debug.depend.file=usr/bin/rpm2cpio
+
+# in OI legacy/retired 64bit packages
+# GDM and Gnome greeter, only gdm-session-worker has to run in 32bit
+#
+# pkg.debug.depend.file does not working yet, there is no appropriate package in the OI repo
+# depend type=require fmri=__TBD pkg.debug.depend.file=usr/sbin/gdm-binary
+depend type=require fmri=system/display-manager/gdm@2.30.7
+# depend type=require fmri=__TBD pkg.debug.depend.file=usr/bin/gnome-session-properties
+depend fmri=gnome/gnome-session@2.32.1 type=require
+# depend type=require fmri=__TBD pkg.debug.depend.file=usr/lib/amd64/libgnome-desktop-3.so
+depend fmri=library/desktop/gnome-desktop@2.32.1 type=require
+# depend type=require fmri=__TBD pkg.debug.depend.file=/usr/libexec/gnome-settings-daemon
+depend fmri=gnome/gnome-settings-daemon@2.32.1 type=require
+depend fmri=library/gnome/libgnomekbd@2.91.92 type=require
+
+# 32bit packages
+# only for loginGUI
+depend fmri=cde/cde-runtime@0.5.11 type=require
+depend fmri=library/tooltalk@0.5.11 type=require
+depend fmri=library/motif@0.5.11 type=require
+
+# gdm-session-worker 32bit libraries
```

Comment:
Could you please elaborate what exactly this means?

Reply:
These are libraries which are used by gdm-seesion-worker.